tests/data/claypit/tests/_support/OtherHelper.php

```diff
@@ -1,12 +1,6 @@
 <?php
 namespace Codeception\Module;
 
-
 class OtherHelper extends \Codeception\Module
```

Comment:
Nope. It is set in corresponding suite config. Sure, this file is empty and does not do anything, but some other empty Helpers of are required in tests to run.